boilerplate/fullstack/remix/app/routes/api.auth.$.tsx

```diff
@@ -0,0 +1,30 @@
+import { json } from "@remix-run/node";
+import { ActionFunctionArgs, LoaderFunctionArgs } from "react-router-dom";
+import handleAuthAPIRequest from "../lib/authAPIRequestHandler.js";
+import { createPreParsedRequest } from "../lib/superTokensHelpers.js";
+
+const handleCall = handleAuthAPIRequest(Response);
+
+// Action function for handling POST requests
+export async function action({ request }: ActionFunctionArgs) {
+    try {
+        const preParsedRequest = createPreParsedRequest(request);
+        const res = await handleCall(preParsedRequest);
+        return res;
+    } catch (error) {
+        return json({ error: "Internal server error" }, { status: 500 });
+    }
+}
+// Loader function for handling GET requests that also adds cache control headers
+export async function loader({ request }: LoaderFunctionArgs) {
+    try {
+        const preParsedRequest = createPreParsedRequest(request);
+        const res = await handleCall(preParsedRequest);
+        if (!res.headers.has("Cache-Control")) {
+            res.headers.set("Cache-Control", "no-cache, no-store, max-age=0, must-revalidate");
+        }
+        return res;
```

Comment:
we should return some JSON information from this API like the user id, session access token paylado (similar to how we have the `/sessioninfo GET` API in our example apps

Reply:
✔️